docs/tutorials/wiki/src/authorization/tutorial/security.py

```diff
@@ -1,5 +1,16 @@
-USERS = {'editor':'editor',
-          'viewer':'viewer'}
+import bcrypt
+
+
+def hash_password(pw):
+    return bcrypt.hashpw(pw.encode('utf-8'), bcrypt.gensalt())
```

Comment:
Just to clarify, we are aware we are returning bytes here, and happy with that? This is slightly different from the wiki2 [1] tutorial which keeps things as unicode strings at all times.

[1] https://github.com/Pylons/pyramid/blob/7ab666e3e2539662ff3324a4bdedbca3c8f8040a/docs/tutorials/wiki2/src/authentication/tutorial/models/user.py#L20-L28